modules/rostests/apitests/shell32/CFSFolder.cpp

```diff
@@ -120,14 +120,14 @@ VOID TestInitialize()
     PERSIST_FOLDER_TARGET_INFO pfti = {0};
     PERSIST_FOLDER_TARGET_INFO queriedPfti;
     hr = ppf3->InitializeEx(NULL, NULL, NULL);
-    ok(hr == (IsWindows7OrGreater() ? E_INVALIDARG : E_OUTOFMEMORY), "hr = %lx\n", hr);
+    ok(hr == (IsWindowsVistaOrGreater() ? E_INVALIDARG : E_OUTOFMEMORY), "hr = %lx\n", hr);
```

Comment:
Let's make it less strict and accept either error code here just like wine tests do: https://github.com/reactos/reactos/blob/master/modules/rostests/winetests/shell32/shlfolder.c#L587

Reply:
Rationale: This makes it easier for us to provide compatibility without apitests shouting at us.